CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ cmake_minimum_required(VERSION 3.8)
 
 project(cmcstl2 CXX)
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+find_package(Sanitizer COMPONENTS address undefined REQUIRED)
+
```

Comment:
Don't we need a change to `test/CMakeLists.txt` to link `Sanitizer::all` into the `stl2_test_config` interface library?

Reply:
Yes, I'll add that.